src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,17 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Random random = new Random();
+
+    public static Color getRandomColor() {
+        Color[] colors = Color.values();
+        int index = random.nextInt(colors.length);
+        return colors[index];
+    }
+
+    public enum Color {
+        RED, GREEN, BLUE, YELLOW, ORANGE, PURPLE, PINK;
     }
 }
```

Comment:
The class `ColorSupplier` should not have a static method `getRandomColor`. Static methods are generally discouraged. Instead, create an instance of `ColorSupplier` and call the method on it.